dark-mode.css

```diff
@@ -505,32 +575,63 @@ body.dark-mode .rcx-options .rcx-option--selected:hover {
 
 /***** Buttons *****/
 
-/* Regular button style */
-body.dark-mode .main-content .rcx-button:not(.rcx-button--ghost) { /* Default */
-	background-color: var(--color-dark) !important;
+/* Default style for buttons */
+
+body.dark-mode .main-content .rcx-button:not(.rcx-button--primary, .rcx-button--danger) {
+	background-color: var(--color-darkest);
 }
-body.dark-mode .main-content .rcx-button:is(.rcx-button--ghost, .rcx-button):hover { /* Hovered or selected */
-	background-color: var(--color-darkest) !important;
+body.dark-mode .main-content .rcx-button:is(.rcx-button--icon) {
+	border-color: transparent !important;
+}
+body.dark-mode .main-content .rcx-button:hover,
+body.dark-mode .main-content .rcx-button.rcx-button--icon-info { /* Replaces :focus rule */
+	background-color: var(--color-dark);
 }
 
-/* Square (icon) button style */
-body.dark-mode .main-content .rcx-button--square:not(.rcx-button--ghost), /* Default */
-body.dark-mode .main-content .rcx-button--square:is(.rcx-button--ghost, .rcx-button):hover {  /* Hovered or selected */
+/* Same style, for unusual buttons/icons/etc */
+/* ----------------------------------------- */
+/*	|	CSS rule					|	Location		|	Description					|
+	|-------------------------------|-------------------|-------------------------------|
+	|	.message-actions__menu-icon	|	Right sidebar	|	Message actions menu items	|
+	|	...							|	...				|	...							|
+*/
+
+body.dark-mode .message-actions__menu-icon {
 	background-color: var(--color-darkest) !important;
+}
+body.dark-mode .message-actions__menu:hover,
+body.dark-mode .message-actions__menu-icon:hover,
+body.dark-mode .message-actions__menu-icon:focus {
+	background-color: var(--color-dark) !important;
 	border-color: transparent !important;
 }
 
-body.dark-mode .main-content .rcx-button--square:is(.rcx-button--ghost, .rcx-button):focus {
-	background-color: var(--color-darkest);
+/* Inverted style, for buttons on dark backgrounds */
```

Comment:
Same as above. If this documentation is helpful but ends up needing to be moved, we may be able to move it to a separate document, potentially CONTRIBUTING.md or a new doc.

Reply:
I find them useful to find what a CSS rule is associated to, especially when classes change over versions, and I'm afraid that moving them to a separate file would complicate the job